api/indexator.rb

```diff
@@ -0,0 +1,53 @@
+require 'open-uri'
+require 'json'
+
+class Indexator
+  BASE = [1988, 1996, 2004, 2013]
+
+  def base_index(base, base_year, base_month)
+    JSON.parse(open("https://fi7661d6o4.execute-api.eu-central-1.amazonaws.com/prod/indexes/#{base}/#{base_year}-#{sprintf('%02i', base_month.month)}").read)["index"]["MS_HLTH_IDX"]
```

Comment:
Your `base_month` should return a month in Integer format (and not a Date) to be consistent with the `current_month` method which returns an Integer
```suggestion
    JSON.parse(open("https://fi7661d6o4.execute-api.eu-central-1.amazonaws.com/prod/indexes/#{base}/#{base_year}-#{sprintf('%02i', base_month)}").read)["index"]["MS_HLTH_IDX"]
```